neurom/view/common.py

```diff
@@ -381,11 +381,8 @@ def plot_style(fig, ax, **kwargs):
         plt.show()  # pragma no cover
```

Comment:
I think this usage of plt. (and the one above) might not work how people expect: if the figure passed in to the function is the 'active' one in matplotlibs internal state, the wrong diagram will be shown...
Any suggestions: @lidakanari or @eleftherioszisis